zap/src/main/java/org/parosproxy/paros/extension/edit/ExtensionEdit.java

```diff
@@ -127,7 +128,7 @@ public static KeyStroke getFindDefaultKeyStroke() {
     /**
      * This method initializes popupMenuFind
      *
-     * @return org.parosproxy.paros.extension.ExtensionPopupMenu
+     * @return org.parosproxy.paros.extension.edit.PopupFindMenu
```

Comment:
Better remove the type as done in `getMenuFind()`, it's redundant (given by the method signature) and does not require further changes when the type changes (like happened here and in the other methods).